rpc/codec/message/arith.proto

```diff
@@ -9,8 +9,9 @@ option go_package = "message";
 import "gogoproto/gogo.proto";
 
 message ArithRequest {
-	optional int32 a = 1 [(gogoproto.nullable) = false];
-	optional int32 b = 2 [(gogoproto.nullable) = false];
+	optional string user = 1 [(gogoproto.nullable) = false];
```

Comment:
Similar to `GossipRequest`, you could add an `arith.go` package and manually implemented `ArithRequest.GetUser()` method.